exchanges/order/order_test.go

```diff
@@ -1000,18 +1000,26 @@ func TestValidationOnOrderTypes(t *testing.T) {
 	}
 
 	cancelMe = new(Cancel)
-	if cancelMe.Validate() != ErrPairIsEmpty {
-		t.Fatal("unexpected error")
+	err := cancelMe.Validate()
+	if !errors.Is(err, nil) {
+		t.Errorf("received '%v' expected '%v'", err, nil)
+	}
+
+	err = cancelMe.Validate(cancelMe.PairAssetRequired())
+	if err == nil || err.Error() != ErrPairIsEmpty.Error() {
```

Comment:
errors.is? and below?

Reply:
Its due to using the Errors type which appends errors but doesn't keep their type. So there is no way to check:
```
    order_test.go:1010: received 'order pair is empty' expected 'order pair is empty'
    order_test.go:1016: received 'order asset type is not set' expected 'order asset type is not set'
```